### ISSUE TITLE
How do I change the background color for images with a transparent background?

### ISSUE THREAD

**Issue by airyng** [Norserium/vue-advanced-cropper#90]
If you set an image with a transparent background, then after processing it, the canvas background will set to black.
how can I change the default color for the background?

Now my component looks like this:
![image](https://user-images.githubusercontent.com/14352077/100874498-55e52800-34ad-11eb-86ed-51c6b503ed18.png)

**Comment by Norserium** [comment]
@airyng, how do you processing your image? Similar issue #87.

**Comment by Norserium** [comment]
By the way, you can set aspect ratio by one property:
```html
<cropper
    :stencil-props="{
         aspectRatio: 1
    }"
/>
```

**Comment by airyng** [comment]
Thanks for response.
I changed max and min props to aspectRatio prop.

About processing. Its divided to 3 steps:

Step 1:
`save () { const { canvas } = this.$refs.cropper.getResult(); this.$emit('onSave', canvas); }`

Step 2:
Resizing image
![image](https://user-images.githubusercontent.com/14352077/100970565-da7f8700-353d-11eb-9d99-ff4c515d9113.png)

Step 3:
Uploading to server
![image](https://user-images.githubusercontent.com/14352077/100970650-01d65400-353e-11eb-9da0-c9445d831b19.png)

I ran a few tests and found out that the real problem is changing mime/type to image/jpeg when converting, as stated in issue # 87 . It turns out that the problem is not in your plugin, but in the nuances of converting types. But then I can't figure out what to do if I don't want to send png to the server. I want to send jpg right away. Is there a way to convert an image with changing the default color for the alpha channel?
